plugin/helpers_i18n.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package plugin
+
+import (
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/mattermost/go-i18n/i18n"
+	"github.com/mattermost/mattermost-server/model"
+)
+
+func (p *HelpersImpl) TServer(translationID string, args ...interface{}) string {
+	if p.localizationSettings == nil {
+		return translationID
+	}
+
+	if p.locales[*p.localizationSettings.DefaultClientLocale] != "" {
```

Comment:
Shouldn't this use the DefaultServerLocale?

Reply:
From the system console: `Default language for newly created users and pages where the user hasn't logged in.` I think that's what we want. The difference between the two is a bit confusing.

Reply:
I guess the difference between the two settings is meaningless now that we aren't translating all of the log messages any more.